pyproject.toml

```diff
@@ -47,7 +47,7 @@ classifiers = {file = ["classifiers.txt"]}
 [project.optional-dependencies]
 glib = ["PyGObject"]
 tornado = ["tornado"]
-trio = ["trio"]
+trio = ["trio>=0.22.0", "exceptiongroup"]
```

Comment:
Please also add `exceptiongroup` to the functional tests requirements. Without it tests fail

Reply:
Whoops, sorry. I'd only run the tests on Python 3.10 so hadn't noticed that they failed on later Python versions. Now pushed a fix.